### ISSUE TITLE
[NBug] Unable to parse object ID "".

### ISSUE THREAD

**Issue by EnochG1** [gitextensions/gitextensions#6644]
## Error Details
```
System.FormatException: Unable to parse object ID "".
   at GitUIPluginInterfaces.ObjectId.Parse(String s)
   at GitCommands.GitModule.GetCurrentCheckout()
   at GitUI.RevisionGridControl.ForceRefreshRevisions()
   at GitUI.CommandsDialogs.FormBrowse.RefreshRevisions()
   at GitUI.GitUIExtensions.<InvokeAsync>d__11.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.ThreadHelper.<>c__DisplayClass13_0.<<FileAndForget>b__0>d.MoveNext()
```

## Environment
- Git Extensions 3.1.0.5877
- Build bb1687f6bd0450d0c1a2bae10a37c65493fc1294
- Git 2.21.0.windows.1
- Microsoft Windows NT 10.0.17134.0
- .NET Framework 4.7.3394.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
Please fill in all required information and ping us back to re-open.

**Comment by RussKie** [comment]
Fixed in 3.1.1

**Comment by gerhardol** [comment]
Duplicate of #6512 
Corrected in #6513, included in 3.2
